install.sh

```diff
@@ -141,6 +146,15 @@ sudo apt-get install llvm-9
 export LLVM_CONFIG=/usr/bin/llvm-config-9
 python3 setup.py build develop --user
 
+###### Install QDTrack ######
+cd $PYLOT_HOME/dependencies/
+git clone https://github.com/mageofboy/qdtrack.git
+git clone https://github.com/open-mmlab/mmdetection.git
+cd mmdetection
+python3 setup.py develop #need to add mmcv
```

Comment:
How reliable is mmdetection's installation? Did it work out of the box for you?

Reply:
unfortunately it is not reliable and installation depends on torch and cuda version